test/compress/issue-427.js

```diff
@@ -0,0 +1,14 @@
+wrap_func_args: {
+    options = {
+        negate_iife: false,
+    }
+    beautify = {
+        wrap_func_args: true,
+    }
+    input: {
+        console.log(function() {
+            return "test";
+        });
+    }
+    expect_exact: 'console.log((function(){return"test"}));'
+}
```

Comment:
Would be cool to add a test which disables negate_iife, and enables `wrap_iife` and `wrap_func_args` by default.

The test would ensure that `console.log((function() { return 42 })())` doesn't add an extra pair of parens.